esp/smc/SMCLib/InfoCacheReader.cpp

```diff
@@ -27,8 +27,20 @@ bool CInfoCache::isCachedInfoValid(unsigned timeOutSeconds)
 
 void CInfoCacheReaderThread::threadmain()
 {
-    PROGLOG("CInfoCacheReaderThread %s started.", name.get());
-    unsigned int autoRebuildMillSeconds = 1000*autoRebuildSeconds;
+    unsigned int autoRebuildMillSeconds;
```

Comment:
how often do we expect on-demand requests?
Are on-demand requests manual, or does eclwatch programmatically requests them?

Reply:
From what I see they are manual in response to navigating to the 'Pods' tab of the 'Topology' page. @jeclrsg could you confirm if the UI calls `/WsCloud/GetPODs` in any other circumstance?

Reply:
> From what I see they are manual in response to navigating to the 'Pods' tab of the 'Topology' page. @jeclrsg could you confirm if the UI calls `/WsCloud/GetPODs` in any other circumstance?

Does ECLWatch periodically refresh that tab?

Reply:
That is the only place in ECL Watch that is calling /WsCloud/GetPODs. And it will only be called when someone navigates to the page or clicks the refresh button.